.github/workflows/main.yml

```diff
@@ -11,8 +11,9 @@ jobs:
 
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
-        python-versions: [3.6, 3.9, 3.11]
+        python-versions: ["3.6", "3.9", "3.12"]
```

Comment:
Since the python 3.11 is still available from RHEL 9.2~, I think we'd better don't remove the `3.11` from the list.
- https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/installing_and_using_dynamic_programming_languages/assembly_installing-and-using-python_installing-and-using-dynamic-programming-languages